specs/multispec.yml

```diff
@@ -0,0 +1,79 @@
+version: 1
+
+specs:
+  distroinfo:
+    centos:
+      distros:
+        - centos-7-x86_64
+      s2i_base: centos/s2i-base-centos7
+      org: "centos"
+      prod: "centos7"
+      logos: "centos-logos"
+      enablerepo: "--enablerepo=centosplus"
+      preinstall_cmd: ['yum install -y centos-release-scl']
+      python_pkgs: ['{{ spec.scl }}', '{{ spec.scl }}-python-devel',
+                    '{{ spec.scl }}-python-setuptools', '{{ spec.scl }}-python-pip']
+      base_pkgs: ['nss_wrapper', 'httpd24', 'httpd24-httpd-devel', 'httpd24-mod_ssl',
+                  'httpd24-mod_auth_kerb', 'httpd24-mod_ldap', 'httpd24-mod_session',
+                  'atlas-devel', 'gcc-gfortran', 'libffi-devel', 'libtool-ltdl enchant']
+      extra_pkgs: ['libjpeg-turbo', 'libjpeg-turbo-devel']
```

Comment:
Since `extra_pkgs` is only used for python27, would it be possible to create a dictionary here? For example:

    extra_pkgs:
      - 2.7: ['libjpeg-turbo', 'libjpeg-turbo-devel']

That way we could hopefully access the variable as `spec.extra_pkgs[spec.version]`.

If this turns out not to work, please rename the variable to be specific about the python version, for example `python27_extra_pkgs`, so the usage is clearer.